talentup/webapp-revamp
# Fixed array bug

### What does this PR do?
- refactor profile forms
- fix array bug

### How can it be manually tested?
- launch website
- go to edit profile
- fill the necessary info and click next
- the error should be fixed
